code/__DEFINES/dcs/signals/signals_object.dm

```diff
@@ -305,9 +305,10 @@
 // Jetpack things
 // Please kill me
 
+//called in datum/component/jetpack/proc/activate() : (source, user)
+#define JETPACK_COMPONENT_ACTIVATION_FAILED (1<<0)
```

Comment:
I'm sorry I know this is taking forever, but I still really don't like the smell of this.
Signals are supposed to be like, generic right? in theory anyone could receive these and say "hey no you can't turn your (jetpack/thruster/modsuit pack) on right now"

I don't like this change because it removes that for cleanliness now, but ends up tightly tying the component to these 3 signals, which impedes future users. (and also leaves the other flags just sitting there which I think was a mistake).

Reply:
It's done. Please lets end this